src/model_config_tests/models/access.py

```diff
@@ -0,0 +1,77 @@
+"""Specific ACCESS-ESM1.5 Model setup and post-processing"""
+
+from pathlib import Path
+from typing import Any
+
+import yaml
+
+from model_config_tests.models.model import SCHEMA_VERSION_1_0_0, Model
+from model_config_tests.models.mom import mom5_extract_checksums
+
+# Default model runtime (24 hrs)
+DEFAULT_RUNTIME_SECONDS = 86400
+
+
+class Access(Model):
```

Comment:
Just for my own understanding, why is `Access` `ACCESS-ESM1.5` specific? It would seem to imply a suite of ACCESS models rather than just ESM1.5.

Reply:
I used `Access` as it is same name of the payu model driver used to run ACCESS-ESM1.5 (so that's the `model: access` in config.yaml). I could name the class differently to make it more clear, so something like `AccessEsm1p5` or `AccessEsm`? @aidanheerdegen would `access` be used for any other models? The payu code just refers to it as `The payu interface for the ACCESS coupled climate model`

Reply:
The driver could be used for other ACCESS models, but currently isn't as far as I know.

These tests are more specific to ESM1.5 I think, in that models can share a driver but have quite different configuration requirements.

So I'd be in favour of making it ESM1.5 specific.

Reply:
I've renamed the model class to `AccessEsm1p5` so it's more clear it's configured for ESM1.5